defs/scenarios/openstack/nova/bugs.yaml

```diff
@@ -1,4 +1,9 @@
 checks:
+  has_1904580:
+    input:
+      path: 'var/log/nova/nova-compute.log'
+    expr: '.+Permissions 0644 for .+ are too open'
+    hint: '0644'
```

Comment:
Should we perhaps make the "0644" a "\d+" so that we catch the message regardless of the current permissions of the file?

Reply:
done

Reply:
done